rotkehlchen/api/v1/encoding.py

```diff
@@ -114,7 +115,11 @@ def _deserialize(
     ) -> Timestamp:
         try:
             timestamp = deserialize_timestamp(value)
-        except DeserializationError as e:
+            # NB: do not include the statement below in 'deserialize_timestamp()'
+            # as the backend deserializes timestamps in milliseconds across the
+            # app (e.g. exchanges) and it would raise an out of range ValueError
```

Comment:
I don't understand what you are doing here. I don't see a point in this change. As I said below this is our API. We set the rules here and the API expects seconds timestamps, not milisecconds.